handlers/clienthandler.go

```diff
@@ -0,0 +1,59 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"tiberious/types"
+
+	"github.com/gorilla/websocket"
+	"gopkg.in/redis.v3"
```

Comment:
I forgot that I included redis in this PR, I was just sort of playing around with it. We should use some form of database to store basic information that way multiple servers can be configured for the same domain/network.